iroh-bytes/src/get.rs

```diff
@@ -445,24 +458,28 @@ pub mod fsm {
             }
         }
 
-        /// Write the entire blob to a slice writer
-        pub async fn write_all<D: AsyncSliceWriter>(
-            self,
-            data: D,
-        ) -> result::Result<AtEndBlob, DecodeError> {
-            self.write_all_with_outboard::<D, D>(None, data).await
+        /// The geometry of the tree we are currently reading.
+        pub fn tree(&self) -> &bao_tree::BaoTree {
```

Comment:
There are a couple of external types that are used in the public api here (BaoTree, blake3::Hash, RangeSet2). Please make sure they are reexported, so it is easy to pull them in in.